### DISCUSSION TITLE
PNG driver doesn't support 5 bands

### DISCUSSION THREAD

**Discussion by MarcYin** [developmentseed/titiler#654] [Q&A]
#### Problem description

[this should explain **why** the current behavior is a problem and why the expected output is a better solution.]

When using the titiler to render remote COGs as tile layer. [One](https://gws-access.jasmin.ac.uk/public/nceo_isp/S2/30/U/XE/S2A_MSIL1C_20200206T111231_N0209_R137_T30UXE_20200206T113812.SAFE/GRANULE/L1C_T30UXE_A024158_20200206T111434/IMG_DATA/BOA_RGB.tif) works and [One](https://gws-access.jasmin.ac.uk/public/nceo_isp/S2/30/U/WD/S2A_MSIL1C_20230404T112111_N0509_R037_T30UWD_20230404T150112.SAFE/GRANULE/L1C_T30UWD_A040646_20230404T112113/IMG_DATA/BOA_RGB.tif) does not.

A xyz tile for the working [one](https://titiler.xyz/cog/tiles/WebMercatorQuad/11/1022/660.png?url=https://gws-access.jasmin.ac.uk/public/nceo_isp/S2/30/U/XE/S2A_MSIL1C_20200206T111231_N0209_R137_T30UXE_20200206T113812.SAFE/GRANULE/L1C_T30UXE_A024158_20200206T111434/IMG_DATA/BOA_RGB.tif)

A xyz tile not working [one](https://titiler.xyz/cog/tiles/WebMercatorQuad/11/1003/631.png?url=https://gws-access.jasmin.ac.uk/public/nceo_isp/S2/30/V/VH/S2B_MSIL1C_20210112T113339_N0209_R080_T30VVH_20210112T123741.SAFE/GRANULE/L1C_T30VVH_A020126_20210112T113447/IMG_DATA/BOA_RGB.tif)
 

#### Expected Output

Both of them should work...

#### Environment Information

Javascript and python

**Comment by vincentsarago** [comment]
The error message is quite meaningful, you're trying to create a PNG tile with 5 bands (4 data bands and 1 alpha band)

Looking at your file: 
https://titiler.xyz/cog/info?url=https://gws-access.jasmin.ac.uk/public/nceo_isp/S2/30/V/VH/S2B_MSIL1C_20210112T113339_N0209_R080_T30VVH_20210112T123741.SAFE/GRANULE/L1C_T30VVH_A020126_20210112T113447/IMG_DATA/BOA_RGB.tif

It's a 4 bands data (none of the band tagged as alpha or mask) so when creating a tile you need to pass band indexes (bidx=1&bidx=2&bidx=3)

**Comment by MarcYin** [comment]
Thanks for the reply. Would it be possible to automatiaclly identify the fourth band as the alpha channel?
